core/src/test/scala/scalacss/RandomData.scala

```diff
@@ -34,16 +34,12 @@ object RandomData {
       LastOfType, Link, OnlyOfType, OnlyChild, Optional, OutOfRange, ReadOnly, ReadWrite, Required, Target, Valid,
       Visited, After, Before, FirstLetter, FirstLine, Selection)
 
-    val needInt = NonEmptyList[Int => Pseudo](
-      NthChild, NthLastChild, NthLastOfType, NthOfType)
-
     val needStr = NonEmptyList[String => Pseudo](
       Custom(_, PseudoClass), Custom(_, PseudoElement), Lang, Not(_))
 
     lazy val self: Gen[Pseudo] =
       Gen.frequency[Pseudo](
         objects.size -> Gen.oneofL(objects),
-        needInt.size -> Gen.oneofL(needInt).flatMap(Gen.positiveint.map),
```

Comment:
What you've done here is completely remove `NthChild, NthLastChild, NthLastOfType, NthOfType` from generatable RandomData.

Could you please add new generators for `NthChild, NthLastChild, NthLastOfType, NthOfType, Attr, AttrExists, Attr...`